libs/base/src/time/since_start.rs

```diff
@@ -25,8 +25,6 @@ prelude! { time::* }
 ///
 /// This type represents a point in time **relative to** the start time of the run of the program
 /// being profiled, which is a [`Date`] (an absolute point in time).
-///
-/// [`Date`]: struct.date.html (The Date struct)
```

Comment:
You should just use the actual Rust path for the type instead of the old html-link style. Module `time` is in scope, and `Date` is in it, so the following should work:

```rust
/// [`Date`]: time::Date
```

Alternatively, rewriting the link itself from `` is a [`Date`] `` to `` is a [`Date`][time::Date] `` should work too.